plugins/heroku/test/setSlug.test.ts

```diff
@@ -28,6 +35,13 @@ jest.mock('@dotcom-tool-kit/state', () => {
 })
 
 describe('setSlug', () => {
+  it('calls setConfigVars for each appIds', async () => {
+    mockHerokuPost.mockImplementationOnce(async () => Promise.resolve(goodHerokuResponse[0]))
+    mockHerokuPost.mockImplementationOnce(async () => Promise.resolve(goodHerokuResponse[1]))
+    await setSlug(logger, slugId)
+    expect(setConfigVars).toBeCalledTimes(2)
```

Comment:
**praise**: good that you're in the habit of writing tests for new code you add, I need to get better at that!

**suggestion**: would be good to check that the mock function is being called with the expected system code
```suggestion
    const systemCode = 'test-app'
    await setSlug(logger, slugId, systemCode)
    expect(setConfigVars).toHaveBeenNthCalledWith(1, logger, 'app-id-1', 'production', systemCode)
    expect(setConfigVars).toHaveBeenNthCalledWith(2, logger, 'app-id-2', 'production', systemCode)
```